### PR TITLE
open-api updated

### PR DESCRIPTION
**Pull request by ahadvl**
Add some new routers to open-api file.

### CODE REVIEW

**Review by al-abd — APPROVED** [review]
@AzerothA important review.

File: open-api.json [hunk 1]
```diff
@@ -208,12 +222,12 @@
...
                 "responses": {
                     "200": {
-                        "description": "Returns the list of all orgs"
+                        "description": "Returns the list of all organizations"
                     }
                 }
             },
             "post": {
-                "description": "Adds the new org",
+                "description": "Add the new organization",
```
Comment (nyzd) [inline open-api.json:230]: No need for 'the' keyword, please remove it. Or reply with the reasoning behind used grammar.

File: open-api.json [hunk 2]
```diff
@@ -267,13 +281,14 @@
                 ],
                 "responses": {
                     "200": {
-                        "description": "This means new org was created"
+                        "description": "This means new organization was created"
```
Comment (nyzd) [inline open-api.json:284]: Please use this term in every response description, or change the grammar to look a like the past descriptions.

File: open-api.json [hunk 3]
```diff
@@ -510,7 +530,225 @@
...
+                                        "example": "f9339c7a-749a-408e-a03e-a22cd87fe3a1"
+                                    },
+                                    "object": {
+                                        "type": "string",
+                                        "description": "The controller that the user need to access",
+                                        "example": "Translation, Reciters & etc"
+                                    },
+                                    "action": {
+                                        "type": "string",
+                                        "description": "The action that the user need to implement on controller",
```
Comment (nyzd) [inline open-api.json]: This sentence makes reader confused. Please alternate it with more simple and understandable sentence.

File: open-api.json [hunk 4]
```diff
@@ -510,7 +530,225 @@
...
+                                        "example": "Add, Edit, View, List, Delet & etc"
+                                    },
+                                    "conditions": {
+                                        "type": "array",
+                                        "description" : "each condition must include name & value, Possible Conditions: isOwner(Boolean)",
```
Comment (nyzd) [inline open-api.json]: You kinda changed the concept of description in this part, before this section used to describe object attribute, but in this case you described the policy of using conditions, please reply with comment if you have an explanation.

File: open-api.json [hunk 5]
```diff
@@ -510,7 +530,225 @@
...
+                                        "example": "[{ \"name\": \"isOwner\", \"value\":\"True\"]"
```
Comment (nyzd) [inline open-api.json]: The value of 'value' attribute must not start with uppercase latter.

File: open-api.json [hunk 6]
```diff
@@ -510,7 +530,225 @@
...
+                    "permission"
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Permission with id you specified was edited"
+                    }
+                }
+            },            
+            "delete": {
+                "description": "Delet permission with id you specified",
```
Comment (nyzd) [inline open-api.json]: Typo: delet -> delete

File: open-api.json [hunk 7]
```diff
@@ -57,6 +66,7 @@
             }
         },
         "/account/verify": {
+            "description": "Return the new user's token",
```
Comment (nyzd) [inline open-api.json]: The 's is overused in user word.

**Review by nyzd — CHANGES_REQUESTED** [review]
Congratulations @ahadvl for you first pull request to Natiq-API, you did a pretty good job at fixing and updating the open-api file however there is some changes I was considering and I wrote comments for you to check, and please start the first word uppercase in every sentence, and don't overuse the 's for example in user, using the 's depends on the context of sentence.

File: open-api.json [hunk 8]
```diff
@@ -510,7 +530,225 @@
...
+                                        "example": "Translation, Reciters & etc"
+                                    },
+                                    "action": {
+                                        "type": "string",
+                                        "description": "The action that needs to be implemented on the controller",
+                                        "example": "Add, Edit, View, List, Delete & etc"
+                                    },
+                                    "conditions": {
+                                        "type": "array",
+                                        "description" : "Array of conditions, each condition is an object includ has name & value, currently only possible condition is isOwner(Boolian)",
```
Comment (nyzd) [inline open-api.json]: Typo Boolean

File: open-api.json [hunk 9]
```diff
@@ -510,7 +530,225 @@
...
+                                        "example": "Translation, Reciters & etc"
+                                    },
+                                    "action": {
+                                        "type": "string",
+                                        "description": "The action that the user need to implement on controller",
+                                        "example": "Add, Edit, View, List, Delete & etc"
+                                    },
+                                    "conditions": {
+                                        "type": "array",
+                                        "description" : "Array of conditions, each condition is an object includ has name & value, currently only possible condition is isOwner(Boolian)",
```
Comment (nyzd) [inline open-api.json]: Typo Boolean

File: open-api.json [hunk 10]
```diff
@@ -510,7 +530,225 @@
...
+                                        "example": "f9339c7a-749a-408e-a03e-a22cd87fe3a1"
+                                    },
+                                    "object": {
+                                        "type": "string",
+                                        "description": "The controller that the user need to access",
+                                        "example": "Translation, Reciters & etc"
+                                    },
+                                    "action": {
+                                        "type": "string",
+                                        "description": "The action that the user need to implement on controller",
```
Comment (nyzd) [inline open-api.json]: The implement word doesn't suit in this sentence.

File: open-api.json [hunk 11]
```diff
@@ -186,13 +199,14 @@
                 ],
                 "responses": {
                     "200": {
-                        "description": "edited"
+                        "description": "User full profile was edited"
```
Comment (nyzd) [inline open-api.json]: 'was' Word is not grammatically correct in this sentence. Please remove it.

**Review by nyzd — CHANGES_REQUESTED** [review]
Please, read comments properly, or give you thoughts about comments as a reply. And there's some new typo.

**Review by nyzd — APPROVED** [review]
Fair enough.

**Pull request merged**